index.html

```diff
@@ -22,7 +22,16 @@
     </ul>
     </nav>
     <!Desktop version end>
-    <nav class="menu-img"><img src="images/Union.svg"></nav>
+    <nav class="menu-nav">
+         <button class="hamburger">
+            <div class="bar"></div>
+            </button>
+        </nav>
+        <nav class="mobile-nav">
+            <a href="" class="tag">Portfolio</a>
+            <a href="" class="tag">About</a>
+            <a href="" class="tag">Contact</a>
```

Comment:
- [ ] Your links aren't working. Please add the corresponding ids for each to redirect the user to the intended part of the page.